Games/types/Mafia/roles/cards/MindShifter.js

```diff
@@ -0,0 +1,55 @@
+const Card = require("../../Card");
+const { PRIORITY_EFFECT_GIVER_DEFAULT } = require("../../const/Priority");
+
+module.exports = class MindShifter extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.meetings = {
+            "Warp Mind": {
+                states: ["Night"],
+                flags: ["voting"],
+                targets: { include: ["alive"], exclude: ["Monsters"] },
+                action: {
+                    labels: ["effect"],
+                    priority: PRIORITY_EFFECT_GIVER_DEFAULT,
+                    run: function () {
+                        this.actor.role.data.insane = this.target;
+                        this.target.queueAlert("You will be driven insane if not visited by a player not aligned with the Monsters!");
+                    }
+                }
+            }
+        };
+
+        this.actions = [
+            {
+                labels: ["effect"],
+                priority: PRIORITY_EFFECT_GIVER_DEFAULT-1,
+                run: function() {
+                    if (this.game.getStateName() != "Night")
+                        return;
+
+                    if (!this.actor.role.data.insane){
+                        return;
+                    }
+
+                    var stillInsane = true;
+                    var visitors = this.getVisitors(this.actor.role.data.insane);
+                    for (let visitor of visitors){
+                        if (visitor.role.alignment != "Monsters")
+                            stillInsane = false;
+                            break;
+                        }
+
+                    if (stillInsane){

```

Comment:
```suggestion
                    const becomesInsane = !visitors.find(visitor => visitor.role.alignment != "Monsters")

                    if (becomesInsane){
```

javascript for loops are banned in 52 states

always use `let`/`const`. never use `var` unless you know you have to (or you learnt javascript in 1999)

https://dev.to/paritho/3-reasons-to-use-var-in-javascript-1hoe
https://softwareengineering.stackexchange.com/questions/274342/is-there-any-reason-to-use-the-var-keyword-in-es6

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#instance_methods